assets/app/styles/_nav.less

```diff
@@ -1,604 +0,0 @@
-
-/* colors light */
-.console-os .navbar-pf {background: #34383C;}
-.console-os .navbar-project,
-.console-os .sidebar {background: #e9eaec;}
-.console-os .sidebar .sidebar-drawer,
-.console-os .nav.nav-sidenav > li > a:hover,
-.console-os .nav.nav-sidenav > li.active > a,
-.console-os .sidebar .nav.nav-sidenav > li.active > a:hover {background-color: #e0e2e4;}
-.console-os .nav.nav-sidenav > li > a:hover {background-color: #d7d9dc}
-
-.console-os .navbar-project,
-.console-os .nav-sidenav > li {border-bottom: 1px solid #d7d9dc;}
-.console-os .nav-sidenav > li > a {color: inherit;}
-
-/* colors dark
-.console-os .navbar-pf {background: #34383C;}
-.console-os .navbar-project,
-.console-os .sidebar {background: #4c5157;}
-.console-os .sidebar .sidebar-drawer,
-.console-os .nav.nav-sidenav > li > a:hover,
-.console-os .nav.nav-sidenav > li.active > a,
-.console-os .sidebar .nav.nav-sidenav > li.active > a:hover {background-color: #636b73;}
-.console-os .nav.nav-sidenav > li > a:hover {background-color: #737c85}
-
-.console-os .navbar-project,
-.console-os .nav-sidenav > li {border-bottom: 1px solid #44494e;}
-.console-os .nav-sidenav > li > a {color: #f2f3f4;}
- */
-
-
- .tile {
```

Comment:
just checking that all these styles were ok to delete, assuming these got moved to _tile.less at some point and were just duplicates?